perception/autoware_tensorrt_yolox/include/autoware/tensorrt_yolox/tensorrt_yolox.hpp

```diff
@@ -280,6 +287,8 @@ class TrtYoloX
 
   // flag whether preprocess are performed on GPU
   bool use_gpu_preprocess_;
+  // GPU id for inference
+  uint8_t gpu_id_;
```

Comment:
```suggestion
  const uint8_t gpu_id_;
```

Please add const and use member initialization list.

Reply:
Done :+1: